elements/pfe-navigation/src/pfe-navigation.js

```diff
@@ -184,7 +190,16 @@ class PfeNavigation extends PFElement {
     this._mobileToggle.removeEventListener("click", this._toggleMobileMenu);
     this._searchToggle.removeEventListener("click", this._toggleSearch);
     this._allRedHatToggle.removeEventListener("click", this._toggleAllRedHat);
+    this._allRedHatToggleBack.removeEventListener("click", this._allRedHatToggleBackClickHandler);
     this.removeEventListener("keydown", this._generalKeyboardListener);
+    // log focused element - for development only
+    this.shadowRoot.removeEventListener(
+      "focusin",
+      function(event) {
```

Comment:
Seems odd that this anon function is written twice, seems like it should be a property on the object so we can refer to it when we add the listener and remove it.